geometry/perspective.hpp

```diff
@@ -11,78 +11,78 @@
 #include "geometry/point.hpp"
 #include "geometry/rp2_point.hpp"
 #include "geometry/sphere.hpp"
+#include "quantities/quantities.hpp"
 
 namespace principia {
 namespace geometry {
 namespace internal_perspective {
 
 using base::BoundedArray;
+using quantities::Length;
 
-template<typename Vector>
-using Segment = std::pair<Point<Vector>, Point<Vector>>;
+template<typename Frame>
+using Segment = std::pair<Position<Frame>, Position<Frame>>;
 
-template<typename Vector>
-using Segments = std::vector<Segment<Vector>>;
+template<typename Frame>
+using Segments = std::vector<Segment<Frame>>;
 
 // A perspective using the pinhole camera model.  It project a point of
 // |FromFrame| to an element of ℝP².  |ToFrame| is the frame of the camera.  In
 // that frame the camera is located at the origin and looking at the positive
 // z-axis.  The x- and y- axis of the camera correspond to those of ℝP².
-template<typename FromFrame, typename ToFrame, typename Scalar,
+template<typename FromFrame, typename ToFrame,
          template<typename, typename> class LinearMap>
```

Comment:
The template parameter LinearMap is no longer used (since `RigidTransformation` implies it is orthogonal), drop it.